src/cli/serve/serve.js

```diff
@@ -201,6 +201,19 @@ export default function (program) {
         }
       }
 
+      // Look for ${filename}.tls.yml which exits in the same directory as the found configs
+      for(const config of opts.config) {
```

Comment:
__question:__ any reason you don't want to just include a couple of additional paths (`config/kibana.tls.yml`, `/etc/kibana/kibana.tls.yml` etc.) to a [predefined config path list](https://github.com/elastic/kibana/blob/master/src/legacy/server/path/index.js#L25) and allow users to rely on `--config` and `KIBANA_PATH_CONF` if they want a custom name?

Reply:
No good reason, this seems reasonable to me.